ors-engine/src/main/java/org/heigit/ors/util/GeomUtility.java

```diff
@@ -142,14 +141,12 @@ public static double metresToDegrees(double metres) {
 	public static double getArea(Geometry geom, boolean inMeters) throws InternalServerException {
 		try {
 			if (inMeters) {
-				if (geom instanceof Polygon) {
+				if (geom instanceof Polygon poly) {
 
 					// https://gis.stackexchange.com/questions/265481/geotools-unexpected-result-reprojecting-bounding-box-to-epsg3035
 					System.setProperty("org.geotools.referencing.forceXY", "true");
 
-					Polygon poly = (Polygon) geom;
-
-					CoordinateReferenceSystem sourceCRS = CRS.decode("EPSG:4326");
+                    CoordinateReferenceSystem sourceCRS = CRS.decode("EPSG:4326");
```

Comment:
lol, that's because of tabs being converted to spaces for line 149. I guess we could leave it as tabs/space is mixed up all over the place anyways and might be resolved withe the .editorconfig

Reply:
I think line endings are mixed up as well…
Introducing the editorconfig in #1492 probably should make sure that all files are up to date with it and could then include the commit-ID of a commit that fixes line endings and whitespace in an `--ignore-revs-file` for any `git blame`?

Reply:
The #1492 did not make sure all files are up to date with it. So i guess we leave This change as is and tackle the repo wide style commit afterwards.